src/main/kotlin/uk/gov/justice/digital/hmpps/bodyscan/service/CreateBodyScanService.kt

```diff
@@ -1,20 +1,24 @@
 package uk.gov.justice.digital.hmpps.bodyscan.service
 
-import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.bodyscan.apiclient.BodyScanPrisonApiClient
+import uk.gov.justice.digital.hmpps.bodyscan.apiclient.model.PersonalCareNeeds
 import uk.gov.justice.digital.hmpps.bodyscan.model.BodyScanDetailRequest
 
 @Service
 class CreateBodyScanService(
-  private val prisonApiClient: BodyScanPrisonApiClient,
+  private val bodyScanPrisonApiClient: BodyScanPrisonApiClient,
 ) {
 
-  fun addBodyScan(prisonNumber: String, detail: BodyScanDetailRequest) {
-    log.info("Received request to add body scan: '$detail' to: '$prisonNumber'")
-  }
-
-  companion object {
-    private val log = LoggerFactory.getLogger(this::class.java)
+  fun addBodyScan(prisonNumber: String, details: BodyScanDetailRequest) {
+    val sentenceDetails = bodyScanPrisonApiClient.getSentenceDetails(prisonNumber)
+    bodyScanPrisonApiClient.addPersonalCareNeeds(
+      sentenceDetails.bookingId,
+      PersonalCareNeeds(
+        date = details.date!!,
+        bodyScanReason = details.reason!!,
+        bodyScanResult = details.result!!
+      )
+    )
```

Comment:
separate story but we should probably raise events for this so we can see usage in app insights/data studio